teddycloud/rootfs/etc/nginx/servers/ingress.conf

```diff
@@ -29,6 +29,7 @@ server {
         sub_filter '.concat("/web",' '.concat("$http_x_ingress_path/web",';
         sub_filter 'apiGetTeddyCloudApiRaw("/api/' 'apiGetTeddyCloudApiRaw("$http_x_ingress_path/api/';
         sub_filter 'apiGetTeddyCloudApiRaw(`/api/' 'apiGetTeddyCloudApiRaw(`$http_x_ingress_path/api/';
```

Comment:
is there an actual need for duplicated filters?

Reply:
they are not duplicate, just different strings ` `` ` vs ` "" `